Sources/Datadog/FeaturesIntegration/CrashReporting/CrashReportingWithRUMIntegration.swift

```diff
@@ -53,16 +61,28 @@ internal struct CrashReportingWithRUMIntegration: CrashReportingIntegration {
             return // Only authorized crash reports can be send
         }
 
-        guard let lastRUMViewEvent = crashContext.lastRUMViewEvent else {
-            InternalMonitoringFeature.instance?.monitor.sdkLogger.error("Attempt to send crash report through RUM integration, but last `RUMViewEvent` is missing")
-            return // This integration requires crash report with associated `RUMViewEvent`
-        }
-
         // The `crashReport.crashDate` uses system `Date` collected at the moment of crash, so we need to adjust it
-        // to the server time before processing. Following use of the current correction is not ideal, but this is the best
-        // approximation we can get.
+        // to the server time before processing. Following use of the current correction is not ideal (it's not the correction
+        // from the moment of crash), but this is the best approximation we can get.
         let currentTimeCorrection = dateCorrector.currentCorrection
 
+        if let lastRUMViewEvent = crashContext.lastRUMViewEvent {
+            sendCrashReportToPreviousSession(crashReport, rumViewEventFromPreviousSession: lastRUMViewEvent, using: currentTimeCorrection)
+        } else {
+            // As this will create a new RUM session, we must respect the user sampling configuration
+            if rumFeatureConfiguration.sessionSampler.isSampled() {
+                sendCrashReportToNewDummySession(crashReport, using: currentTimeCorrection)
```

Comment:
```suggestion
        } else if rumFeatureConfiguration.sessionSampler.isSampled() {
                // As this will create a new RUM session, we must respect the user sampling configuration
                sendCrashReportToNewDummySession(crashReport, using: currentTimeCorrection)
```